.github/workflows/ci.yml

```diff
@@ -229,6 +230,14 @@ jobs:
 
           # build docs
           cargo +${{ env.DOCS_RUST_VERSION }} doc --target ${{ matrix.target }} --all-features --no-deps
+      - name: check private item docs
+        if: matrix.kind == 'native'
+        shell: bash
+        run: |
+          set -e
+
+          # wgpu_core package
+          cargo +${{ env.DOCS_RUST_VERSION }} doc --target ${{ matrix.target }} --all-features --no-deps --package wgpu-core --document-private-items 
```

Comment:
TOO MUCH WORK

Reply:
That is, I doubt the other crates build without errors. I'll add more packages once I get this working. Naga is probably in the best shape, because I did test with `--document-private-items` until recently, I think.